src/main/java/com/zendesk/maxwell/zk/ZkService.java

```diff
@@ -0,0 +1,134 @@
+package com.zendesk.maxwell.zk;
+
+import org.I0Itec.zkclient.IZkStateListener;
+import org.I0Itec.zkclient.ZkClient;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.zookeeper.Watcher;
+import org.apache.zookeeper.data.Stat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.UUID;
+
+
+public class ZkService {
```

Comment:
if zookeeper is what we want to couple maxwell to for leader election, would it be better to make use of an apache curator recipe for that purpose?